### ISSUE TITLE
Question: how to make featured-slide one of the sub-slides?

### ISSUE THREAD

**Issue by ericmjl** [webpro/reveal-md#206]
Thank you for building this, @webpro! 

I have a small question, not sure if this is possible or not (but it's okay if it's not currently implemented) - I have a sub-slide that contains a really pretty image, which I would like to make as my featured slide. Using reveal-md, how can I make this happen?

Here is what I have tried:

```
reveal-md slides.md --static docs --disable-auto-open --css custom.css --theme white --absolute-url https://ericmjl.github.io/curve-fitting-talk --featured-slide 6-2
```

I have a separate question, regarding where the custom CSS assets are placed, and I will post it as a second question here on the issue tracker.

Thank you for your time, if you're too busy to handle this, no worries, I totally understand (fellow open source maintainer myself).

**Comment by webpro** [comment]
Hi @ericmjl, thanks! Featuring a sub-slide is currently not supported indeed. It shouldn't be too hard to implement, though:

https://github.com/webpro/reveal-md/blob/2becce4b3544139fca4f226f86b8c26d70e79206/lib/featured-slide.js#L37-L39

**Comment by webpro** [comment]
Thanks to @kazigk, I just released this in [v2.3.0](https://github.com/webpro/reveal-md/releases/tag/2.3.0).
